README.md

```diff
@@ -1,10 +1,45 @@
 # Theia
 
-Theia contains Network Flow Visibility functions which are exacted from the
-[Antrea main repo](https://github.com/antrea-io/antrea). While flow exporter and
-flow aggregator are kept in the Antrea repo, other flow visibility modules, such
-as ClickHouse and Grafana related ones, will be moved to this new repo.
-
-During the code migration period, we will still keep related functions available
-and stable in the Antrea repo, and the functions in this new repo won't be ready
-until we announce the migration is completed.
+Theia is a network observability and analytics platform for Kubernetes. It is
+built on top of [Antrea](https://github.com/antrea-io/antrea), and consumes
+[network flows exported by Antrea](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md)
+to provide fine-grained visibility into the network communications in a
+Kubernetes cluster.
+
+## Getting Started
+
+Getting started with Theia is simple. You can follow the [Getting Started](docs/network-flow-visibility.md#getting-started)
+guide to install Theia and start rocking!
+
+Theia supports network flow visualization and monitoring with Grafana. Check the
```

Comment:
Right now, Theia only supports flow visibility and NP recommendation for a single cluster. Maybe we should highlight this before we can support more?

Reply:
I feel no need to mention not implemented features in README (e.g. we can have a ROADMAP doc for that if we want). There are other K8s visibility solutions support a single cluster only too. Do not think multi-cluster support is a must.